### ISSUE TITLE
Rename of module-name.yang to module-name@revision.yang

### ISSUE THREAD

**Issue by robshakir** [YangModels/yang#75]
Hi,

[This commit](https://github.com/YangModels/yang/commit/ead44523cefc2d0f029056f61253c3b36ab38da8) renames a set of the standard IETF modules (e.g., `inet-inet-types.yang`) and replaces them with `module-name@revision.yang` (for example, https://github.com/YangModels/yang/commit/ead44523cefc2d0f029056f61253c3b36ab38da8#diff-370faa18a85ed465f9c3033c8a5173a5). 

This means that any tool that was using the reference to the base file (i.e., does not care what revision it is importing) is now unable to retrieve these modules.

Was this intended such that these tools should be updated?

Thanks,
r.

**Comment by wlupton** [comment]
Rob,

Does commit 3d518d5bfe1e1d91394c8e1fbb18fea4e52fd5a8 address your concern?

Previously I observed that the latest revision was called `module-name.yang` and previous revisions were called `module-name@revision.yang`. Now all revisions are called `module-name@revision.yang` and `module-name.yang` is a soft link to the latest revision. That seems to work with all the tools that I use (which isn't many: `pyang` and `confdc`/`yanger`).

Thanks,
William

**Comment by robshakir** [comment]
@wlupton 

For those specific modules, it does fix my issue. I remarked to @bclaise that I've already forked the repo to be able to fix this for our internal tooling.

I'd imagine that the YANG catalogue [0] should be something that can fix this problem, since we should be able to link to a specific URL for a file in that, such that tools can refer to the entry there, rather than a specific file/URL.

Cheers,
r.

[0]: https://tools.ietf.org/html/draft-openconfig-netmod-model-catalog-01
